### PR TITLE
Server integrate

### PR DESCRIPTION
**Pull request by myidsan**
Updated quiz logic while integrating with front. They are talking.

### CODE REVIEW

**Comment by myidsan** [comment]
fix #38

File: db_update_scripts/card_update.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import gspread
+import json
+import requests
+import sys
+from oauth2client.service_account import ServiceAccountCredentials
```
Comment (jameswhang) [inline db_update_scripts/card_update.py]: yaaaay more copy pastas

Comment (jameswhang) [reply db_update_scripts/card_update.py]: but it's just some script so i guess this is OK

File: db_update_scripts/quiz_update.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import gspread
+import json
+import requests
+import sys
```
Comment (jameswhang) [inline db_update_scripts/quiz_update.py]: wow more??

File: db_update_scripts/quizcardlist_update.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import gspread
+import json
+import requests
+import sys
+from oauth2client.service_account import ServiceAccountCredentials
+
+def spreadToJSON():
```
Comment (jameswhang) [inline db_update_scripts/quizcardlist_update.py]: MORE????

Comment (myidsan) [reply db_update_scripts/quizcardlist_update.py]: lol these were to upload on local db to test integration :)

**Pull request merged**